components/LessonsButton/LessonsButton.module.css

```diff
@@ -0,0 +1,19 @@
+.btn {
+  border: none;
+  background-color: #959595;
+  color: #f2f2f2;
+  padding: 1em;
+  margin-right: 1px;
+}
+.btn:last-child {
+  margin-right: 0;
```

Comment:
This works well, but there is a good alternative to achieve a similar gap between your buttons without any `margin`.
You can add `grid-column-gap` to your container and remove all margins.